zebra-state/src/service/finalized_state/disk_format/transparent.rs

```diff
@@ -416,36 +416,36 @@ impl AddressTransaction {
         }
     }
 
-    /// Create an [`AddressTransaction`] which starts iteration for the supplied
+    /// Create a range of [`AddressTransaction`]s which starts iteration for the supplied
     /// address. Starts at the first UTXO, or at the `query_start` height,
-    /// whichever is greater.
+    /// whichever is greater. Ends at the last transaction index for an [`AddressLocation`].
     ///
-    /// Used to look up the first transaction with
-    /// [`ReadDisk::zs_next_key_value_from`][1].
+    /// Used to look up transactions with
+    /// [`DiskDb::zs_range_iter`][1].
     ///
     /// The transaction location might be invalid, if it is based on the
```

Comment:
This part of the docs needs updating because there are two locations now: the start can be invalid, the end is always invalid.

Reply:
Updated in https://github.com/ZcashFoundation/zebra/pull/7732/commits/cb9a1f66c53758597329b2f36278df3c8126b818.